docs/standard/garbage-collection/includes/disposables-and-dependency-injection.md

```diff
@@ -0,0 +1,12 @@
+---
+author: IEvangelist
```

Comment:
Most of our include files live in the includes directory at the root of the repo. Also, it's probably not a good idea to have relative links in an include file, since they depend on the including file being in a specific location.

Reply:
In the Azure docs, we started scoping include directories local to their relevant content area. This makes it a lot easier to manage. As for relative links, are you suggesting that we use absolute paths instead - or what alternative specifically?

Reply:
I see there's an includes directory under docs/azure and docs/iot. If we put one here, I think it should go up a level to docs/standard. @BillWagner Are you okay with that?

> As for relative links, are you suggesting that we use absolute paths instead - or what alternative specifically?
Yes, use a path that starts with /dotnet/.

Reply:
I'm curious why you're want to move it to `docs/standard`? What is the benefit of that, I've always favored locally scoped includes - and I do not anticipate this include being needed elsewhere.

Reply:
That was to match the level of the includes folders in the docs/azure and docs/iot directories.

Reply:
Ah, I see. I do not believe it is necessary with this case, unlike those directories - this include is really scoped to the articles at this level.